### ISSUE TITLE
syncfusion_flutter_datepicker/example build fails!

### ISSUE THREAD

**Issue by ajaygautam** [syncfusion/flutter-widgets#64]
flutter run failed:
```
agautam@ajmbp ~/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example (master) $ flutter run
Running "flutter pub get" in example...                             1.5s
Launching lib/main.dart on iPhone 11 Pro in debug mode...
 
Project base configurations detected, removing.
Running Xcode build...                                                  
Xcode build done.                                            5.0s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    error: /Users/agautam/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support
    Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths (in target 'Runner' from project 'Runner')
    error: /Users/agautam/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support
    Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths (in target 'Runner' from project 'Runner')
    error: /Users/agautam/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support
    Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths (in target 'Runner' from project 'Runner')
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.2.99. (in target
    'Runner' from project 'Runner')

Could not build the application for the simulator.
Error launching application on iPhone 11 Pro.
```
my flutter setup:
```
agautam@ajmbp ~/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example (master) $ flutter doctor -v
[✓] Flutter (Channel stable, 1.22.4, on Mac OS X 10.15.7 19H15 darwin-x64, locale en-US)
    • Flutter version 1.22.4 at /Users/agautam/tools/flutter
    • Framework revision 1aafb3a8b9 (9 days ago), 2020-11-13 09:59:28 -0800
    • Engine revision 2c956a31c0
    • Dart version 2.10.4

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/agautam/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /Users/agautam/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.2, Build version 12B45b
    • CocoaPods version 1.10.0

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] Connected device (1 available)
    • iPhone 11 Pro (mobile) • 70B5851E-D6FE-4D5F-8703-D9AEB22DE51E • ios • com.apple.CoreSimulator.SimRuntime.iOS-14-2 (simulator)

! Doctor found issues in 1 category.
```

XCode version 12.2
MacBook: Early 2015 MacOS Catalina v 10.15.7

**Comment by ajaygautam** [comment]
pod install seems to have fixed it:

```
agautam@ajmbp ~/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example (master) $ cd ios/
agautam@ajmbp ~/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example/ios (master) $ pod install
Analyzing dependencies
Downloading dependencies
Installing Flutter (1.0.0)
Generating Pods project
Integrating client project
Pod installation complete! There is 1 dependency from the Podfile and 1 total pod installed.

[!] Automatically assigning platform `iOS` with version `8.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

[!] CocoaPods did not set the base configuration of your project because your project already has a custom config set. In order for CocoaPods integration to work at all, please either set the base configurations of the target `Runner` to `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` or include the `Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig` in your build configuration (`Flutter/Release.xcconfig`).
agautam@ajmbp ~/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example/ios (master) $ echo $?
0
agautam@ajmbp ~/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example/ios (master) $ cd ..
agautam@ajmbp ~/workspace/tmp/flutter-widgets/packages/syncfusion_flutter_datepicker/example (master) $ flutter run
Launching lib/main.dart on iPhone 11 Pro in debug mode...
 
Running Xcode build...                                                  
 └─Compiling, linking and signing...                        18.1s
Xcode build done.                                           35.9s
Waiting for iPhone 11 Pro to report its views...                    11ms
Syncing files to device iPhone 11 Pro...                           206ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
An Observatory debugger and profiler on iPhone 11 Pro is available at: http://127.0.0.1:56862/qUdAXbnVLmo=/
```

**Comment by Indumathi1195R** [comment]
Hi @ajaygautam 

We hope that issue has been resolved at your end. Please let us know if you need further assistance.   

Regards,
Indumathi R
